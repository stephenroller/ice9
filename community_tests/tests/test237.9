# forward w/o proc										fail

forward 
	foo
	(
	a
	: 
	int
	) 
	: 
	int

;
