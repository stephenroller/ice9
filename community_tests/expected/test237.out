line 4: proc foo has no body